llrt_utils/src/encoding.rs

```diff
@@ -63,6 +64,7 @@ impl Encoder {
             Self::Base64 => Ok(bytes_to_b64_string(bytes)),
             Self::Utf8 | Self::Windows1252 => bytes_to_string(bytes, lossy),
             Self::Utf16le => bytes_to_utf16le_string(bytes, lossy),
+            Self::Utf16be => bytes_to_utf16be_string(bytes, lossy),
```

Comment:
I think we can get rid of functions that are just wrapped by calling `bytes_to_utf16_string` directly.